### DISCUSSION TITLE
Test-network-k8s supports couchdb as data base.

### DISCUSSION THREAD

**Discussion by SamYuan1990** [hyperledger/fabric-samples#633] [Ideas]
Test-network-k8s supports couchdb as data base.

**Comment by mbwhite** [comment]
Yes - reasonable idea.... would be a great contribution if anybody wanted to try.

**Reply by SamYuan1990** [reply]
https://github.com/hyperledger/fabric-samples/pull/651 FYI
I made a try with PR above.
